### DISCUSSION TITLE
Tooltip functionality not working for my charts.

### DISCUSSION THREAD

**Discussion by youfiElwy** [XCharts-Team/XCharts#336] [Q&A]
Hey everyone, so i just added a line chart into my unity scene, and the tooltip functionality is working perfectly! Even though i have to admit that i could not get it to work at the start, and all of a sudden it started working on its own, so i just went with it.

I am now working with another chart (a bar chart) and i am also trying to get the tooltip functionality to work, but I am struggling again with it.

I don't know why it is not working. I checked both visualizations (the one where it works, and the one where it doesn't work). They both have the exact same settings and properties. but for some reason the tooltip is not working in my chart. 

Can someone please help! I'm in need of help really fast!!!

I can send screenshots and further details if needed.
youssef47009@gmail.com
You can contact me if you want.

**Comment by monitor1394** [comment]
You can first check if the UI's EventSystem is functioning correctly.
